robboat/templates/homepage.html

```diff
@@ -10,8 +10,7 @@
   <p><a href="https://github.com/robb-oat/server/issues/2"><img src="{% static 'demo.gif' %}"></a></p>
 
   <p><a href="https://github.com/robb-oat/server/issues/2">Preview Robbʼs work</a>. ☝️</p>
-  <p class="cta">👉 <a href="https://github.com/marketplace/robb-oat">“Hire” Robb for free</a>!</p>
-  <h4>User Instructions</h4>
+  <p class="cta">👉 <a href="https://github.com/marketplace/robb-oat">“Hire” Robb for free</a>!</p>  <h2>User Instructions</h2>
```

Comment:
```suggestion
  <p class="cta">👉 <a href="https://github.com/marketplace/robb-oat">“Hire” Robb for free</a>!</p>  
  <h2>User Instructions</h2>
```

You forgot a line break 😎

Reply:
Robb's not great at whitespace, no lie. Thanks for the review! ;-)